pom.xml

```diff
@@ -6,14 +6,61 @@
         <groupId>org.springframework.boot</groupId>
         <artifactId>spring-boot-starter-parent</artifactId>
         <version>2.3.2.RELEASE</version>
-        <relativePath/> <!-- lookup parent from repository -->
+        <relativePath/>
     </parent>
+
     <groupId>me.minidigger</groupId>
     <artifactId>hangar</artifactId>
     <version>0.0.1-SNAPSHOT</version>
+
     <name>hangar</name>
     <description>Ore, but in Spring, but for Paper</description>
 
+    <properties>
+        <java.version>11</java.version>
+
+        <!-- compiling -->
+        <maven.compiler.source>${java.version}</maven.compiler.source>
+        <maven.compiler.target>${java.version}</maven.compiler.target>
+        <maven.compiler.testSource>${java.version}</maven.compiler.testSource>
+        <maven.compiler.testTarget>${java.version}</maven.compiler.testTarget>
+
+        <!-- encoding -->
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+
+        <!-- dependencies -->
```

Comment:
Can you please sort out the webjars to their own section?

Reply:
Done. Also moved the bom version to an own area maybe some more boms will be used in the future.